LoRaMac-node/README.md

```diff
@@ -1,37 +1,41 @@
 
-In this [fork](https://github.com/TheThingsNetwork/LoRaMac-node/) we add support for ATECC608A-TNGLORA as a HW secure element.
+The [LoRaMac-node](https://github.com/Lora-net/LoRaMac-node) `develop` branch supports ATECC608A-TNGLORA as a HW secure element.
 
-This addition means that (theoretically) all the devices/boards supported by LoRaMac-node now support using ATECC608A-TNGLORA as an external HW secure element.
+This support means that (theoretically with the correct I2C config) all the devices/boards supported by LoRaMac-node now support using ATECC608A-TNGLORA as an external HW secure element.
 
 ## Supported Targets
 
 The ATECC608A-TNGLORA communicates via the I2C bus, this means with a proper I2C initialization any microcontroller can communicate with the ATECC608A-TNGLORA.
 
 ### Microchip SAMR34 Support
 
-In this [branch](https://github.com/TheThingsNetwork/LoRaMac-node/tree/feature/SAMR34%2BATECC608A) we add  I2C support for the [SAMR34 Xplained Pro Evaluation Kit](https://www.microchip.com/DevelopmentTools/ProductDetails/dm320111) board based on the existing SAML21 support.
+In the `develop` branch we add  I2C support for the [SAMR34 Xplained Pro Evaluation Kit](https://www.microchip.com/DevelopmentTools/ProductDetails/dm320111) board.
 
 > Make sure to set `BOARD` to `SAMR34` in the CMAKE building system.
 
 ### Other Targets
 
-Since LoRaMac-node is a bare-metal platform, you can adapt any [LoRaMac-node supported platform](https://github.com/TheThingsNetwork/LoRaMac-node/#supported-platforms) to use new or existing I2C pins and communicate with the ATECC608A-TNGLORA similar to the SAMR34.
+Since LoRaMac-node is a bare-metal platform, you can adapt any [LoRaMac-node supported platform](https://github.com/Lora-net/LoRaMac-node/#supported-platforms) to use new or existing I2C pins and communicate with the ATECC608A-TNGLORA similar to the SAMR34.
 
 ## Supported Applications
 
-All applications allow for joining over OTAA via `OVER_THE_AIR_ACTIVATION` flag in each application `Commissioning.h`.
+All applications are supported, as by default, all applications join the network via OTAA (`OVER_THE_AIR_ACTIVATION` flag `Commissioning.h`)
 
 ## Getting Started
 
 1. Get familiar with [LoRaMac-node repository](https://github.com/TheThingsNetwork/LoRaMac-node/#introduction)
 2. Follow the detailed [Usage](https://github.com/TheThingsNetwork/LoRaMac-node/#usage) instructions
-3. To test with SAMR34 target, checkout `feature/SAMR34+ATECC608A`[branch](https://github.com/TheThingsNetwork/LoRaMac-node/tree/feature/SAMR34%2BATECC608A)
-
+3. To test with SAMR34 target, checkout the [LoRaMac-node](https://github.com/Lora-net/LoRaMac-node) `develop` branch.
+	- ```$ git clone https://github.com/Lora-net/LoRaMac-node.git```
+	- ```$ cd LoRaMac-node```
+	- ```$ git checkout develop```
+	- ```$ git submodule update --init --recursive```
```

Comment:
Oneliner;

```bash
$ git clone --branch develop --recurse-submodules https://github.com/Lora-net/LoRaMac-node.git
```

Also no need for triple backtick on single line literal

Reply:
Awesome, thanks for the tip, looks better now!